data/json/terrain.json

```diff
@@ -868,6 +915,7 @@
         "type" : "terrain",
         "id" : "t_palisade",
         "name": "palisade wall",
+        "description": "An age old type of fortification consisting of hefty lumber, staked into the ground and cabled together.",
```

Comment:
Should be "age-old"

Also, no need for a comma here ("staked into the ground" is not the second item in the list – it directly relates to the subject, "hefty lumber")